semantic_conventions/trace/database.yaml

```
@@ -204,6 +204,15 @@ groups:
           The database statement being executed.
         note: The value may be sanitized to exclude sensitive information.
         examples: ['SELECT * FROM wuser_table', 'SET mykey "WuValue"']
+      - id: values
```

```suggestion
      - id: bind_values
```

Not sure if we should be more specific, but I feel just `values` is a bit vague. SQL Server an Oracle call these Bind Variables/Parameters [Oracle](https://docs.oracle.com/cd/A81042_01/DOC/sqlplus.816/a75664/ch34.htm) [Microsoft](https://learn.microsoft.com/en-us/sql/relational-databases/native-client-odbc-queries/using-statement-parameters-binding-parameters?view=sql-server-ver16) so maybe `bind_values` is clearer?

---

I think `db.bind_values` is a lot better. `db.values` can be anything.
`db.statement.values` would be clear and consistent but since we already have `db.statement` as an attribute we can't re-use it as a "namespace".

---

I know what it means for SQL statements, but wonder if there are also values for no-sql statements, like mongo / redis etc?

Does it makes sense to capture under the `sql` namespace? e.g. similar to the existing attribute `db.sql.table` we can have maybe `db.sql.bind_values` or `db.sql.statement.values`?

---

>I know what it means for SQL statements, but wonder if there are also values for no-sql statements, like mongo / redis etc?

Good point. Also, In some dbs, the values have associated "keys", so we could also think in recording those.

---

seems Cassandra CQL supports named bind_markers, see [here](https://cassandra.apache.org/doc/4.1/cassandra/cql/definitions.html#terms)

Maybe this answers also the question if something like this is available outside of SQL.

---

I will close this as resolved, as the new has changed. @Flarna I think your comment makes it also clear keeping at the database namespace makes sense. Feel free to start a new thread @blumamir if you think this still doesn't make sense.